nfd-worker-daemonset.yaml.template

```
@@ -2,29 +2,31 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   labels:
-    app: node-feature-discovery
-  name: node-feature-discovery
+    app: nfd-worker
+  name: nfd-worker
 spec:
   selector:
     matchLabels:
-      app: node-feature-discovery
+      app: nfd-worker
   template:
     metadata:
       labels:
-        app: node-feature-discovery
+        app: nfd-worker
     spec:
       hostNetwork: true
-      serviceAccount: node-feature-discovery
       containers:
         - env:
           - name: NODE_NAME
             valueFrom:
               fieldRef:
                 fieldPath: spec.nodeName
           image: quay.io/kubernetes_incubator/node-feature-discovery:v0.3.0
-          name: node-feature-discovery
+          name: nfd-worker
+          command:
+            - "nfd-worker"
           args:
             - "--sleep-interval=60s"
+            - "--server=$(NFD_MASTER_SERVICE_HOST):$(NFD_MASTER_SERVICE_PORT)"
```

```suggestion
            - "--server=${NFD_MASTER_SERVICE_HOST}:${NFD_MASTER_SERVICE_PORT}"
```
Shouldn't those be curly brackets? I do not think you meant command substitution here. Those are the env vars of the service.

---

Env variable substitution inside k8s spec works with "normal" round brackets